packages/gestalt/src/Dropdown.js

```diff
@@ -279,6 +291,7 @@ export default function Dropdown({
       shouldFocus
       size="xl"
       __dangerouslySetMaxHeight={maxHeight}
+      __onPositioned={() => setIsPopoverPositioned(true)}
```

Comment:
we could merge this for now, but I think I'd be better to use some InternalPopover rather than the exposed Ppopover here in the next PRs... we can keep some props internal in the InternalPopover and we don't have to expose them.